Tipstaff/Controllers/ApplicantController.cs

```diff
@@ -55,10 +57,10 @@ public ActionResult Details(string id)
         public ActionResult Create(string id)
         {
             ApplicantCreationModel model = new ApplicantCreationModel(id);
-            TipstaffRecord tipstaff = _applicantPresenter.GetTipstaffRecord(id);
+            model.tipstaffRecord = _applicantPresenter.GetTipstaffRecord(id);
```

Comment:
shouldn't this "GetTipstaffRecord" method be implemented on the TipstaffRecord Presenter instead of the applicant?